test/javascript/node/console_test.gleam

```diff
@@ -0,0 +1,37 @@
+import plinth/javascript/console
+
+pub fn log_test() {
```

Comment:
How do these tests assert that they are doing the correct thing?

Reply:
Visually. I'd need a way to read stdout otherwise. I don't know a way to test console output.

If you have any pointers on where to look for something like that, I'd be happy to learn and update the tests.

Reply:
I don't think I would write tests at all in this situation. 
If you think there's a think there's a need for a better way, asking in the discord is always full of interesting opinions

Reply:
I needed a way to practically ensure it works, but we can drop them, as I already got what I need. 

I looked into reading stdout in the meanwhile. Interception, piping into a second executable etc. All recipes for brittle tests 😁 . Totally not worth it for a simple bridge FFI.

Reply:
Just dropped the tests…